login.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+
+<link rel="stylesheet" type="text/css" href="css/login.css">
+
+</head>
+<body>
+
+<div id="external">
+<div id="login">
+	<h4>Interview Feedback</h4>
+	<form method="POST">
+		<input type="text" name="userName" placeholder="User Name"> <br>
+		<input type="password" name="pw" placeholder="Password"> <br>
+		<input type="submit" name="login" value="Login">
+	</form>
+</div>
+</div>
+
+<div id="fb1" class:"clearfix">
+    <img src="assets/newfeedback.jpg">
+    <h1>Submit new feedback</h1>
```

Comment:
Why the `h1` here? FYI, a SEO best practice indicates that only one top-level heading should ever be used per page. That is so that tools such as web crawlers can easily infer the main topic of a particular document and use it for presentation purposes in search results.